src/main/scala/chiseltest/ChiselScalatestTester.scala

```diff
@@ -9,13 +9,14 @@ import firrtl.AnnotationSeq
 import org.scalatest._
 import org.scalatest.exceptions.TestFailedException
 import internal.WriteVcdAnnotation
-import logger.Logger
 
 import scala.util.DynamicVariable
 
 trait ChiselScalatestTester extends Assertions with TestSuiteMixin with TestEnvInterface { this: TestSuite =>
 
-  class TestBuilder[T <: MultiIOModule](val dutGen: () => T, annotationSeq: AnnotationSeq, flags: Array[String]) {
+  class TestBuilder[T <: MultiIOModule](val dutGen:        () => T,
```

Comment:
what changed here other than formatting? did we reach consensus on this style?

Reply:
The introduction of `val`s made this line too long.  And although we have not nailed down our formatting style. This formatting is the one we were considering adopting.

Reply:
Not a fan of the one argument per line style, so unless that was one of the style points for which there was a consensus, I'd prefer it be backed out,

Reply:
backed it out